src/components/Frame/Frame.tsx

```diff
@@ -135,6 +135,9 @@ class FrameInner extends PureComponent<CombinedProps, State> {
           classNames={navTransitionClasses}
         >
           <div
+            aria-modal
+            role="dialog"
```

Comment:
This makes sense on mobile but does it make things confusing at all when the nav isn't hidden?

Reply:
Nice catch, I thought this was mobile only markup. I made a change to conditionally apply the attributes & added tests for it.